sample/build.gradle

```diff
@@ -16,8 +16,17 @@ android {
         versionName "1.0"
     }
     buildTypes {
+        debug {
+            // remove unused classes and to make sure the proguard config is correct
+            // https://developer.android.com/studio/build/shrink-code.html#shrink-code
+            minifyEnabled true
+            // don't enable proguard, it doesn't work with instant run
+            // https://developer.android.com/studio/build/shrink-code.html#gradle-shrinker
+            useProguard false
```

Comment:
This is very strange. Because this config don't create the output of the mapping files etc.
But it is implemented like documented .. so 🤷‍♂️